proto/toit/api/organization.proto

```diff
@@ -22,6 +23,7 @@ service OrganizationService {
   rpc GetAPIKeySecret(GetAPIKeySecretRequest) returns (GetAPIKeySecretResponse) {}
 
   rpc UpdateOrganization(UpdateOrganizationRequest) returns (UpdateOrganizationResponse) {}
+  rpc CreateOrganization(CreateOrganizationRequest) returns (CreateOrganizationResponse) {}
```

Comment:
Rename to `CreateNewOrganization` and copy the request and response over here.
The request should not specify the user (that is the current user).